### PR TITLE
Restructure complete. Tidied up a few files, all in working order

### PR DESCRIPTION
**Pull request by dlouise64**
Configured babelrc to allow us to use arrow functions.
Arrow functions will now replace the use of binding methods
Testing env setup will update in another branch (feature/tests)
Components are now located in the components directory, each component filename takes the Classname
Cleaned up variables for props
Webpack removed as had issues bundling with Leaflet

### CODE REVIEW

File: src/components/SearchResults.js [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 class SearchResults extends React.Component {
   constructor(props) {
     super(props);
     this.resourceId = '35de6bf8-b254-4025-89f5-da9eb6adf9a0';
-    this.toggleShowMap = this.toggleShowMap.bind(this);
     this.state = {results: [], show_map: false};
-    this.fetchResults = this.fetchResults.bind(this);
   }
-  toggleShowMap() {
+  toggleShowMap = () => {
```
Comment (Br3nda) [inline src/components/SearchResults.js]: why do we do this syntax? i've not seen it before.

File: src/components/SearchResults.js [hunk 2]
```diff
@@ -66,13 +64,17 @@ class SearchResults extends React.Component {
     );
   }
   renderMap() {
+    const {
+      longitude,
+      latitude
+    } = this.props;
```
Comment (Br3nda) [inline src/components/SearchResults.js]: this seems to be more lines.. but is it a standard way now?

**Comment by Br3nda** [comment]
uglify seems unhappy on travis
```
> uglifyjs-webpack-plugin@0.4.6 postinstall /home/travis/build/ServiceInnovationLab/FamilyServiceDirectorySearchinator/node_modules/react-scripts/node_modules/webpack/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js
module.js:340
    throw err;
          ^
Error: Cannot find module 'sync-exec'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/travis/build/ServiceInnovationLab/FamilyServiceDirectorySearchinator/node_modules/react-scripts/node_modules/webpack/node_modules/uglifyjs-webpack-plugin/lib/post_install.js:9:14)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
```

File: src/components/SearchResults.js (hunk 1 above)
Comment (dlouise64) [reply src/components/SearchResults.js]: @Br3nda This is an arrow function from ES2015 which is transpiled with babel. It allows to avoid binding this.

**Pull request merged**